src/site/facilities/health_service.drupal.liquid

```diff
@@ -19,7 +19,7 @@
                 {% endif %}
 
                 {% if healthService.fieldLocalHealthCareService.length > 0 %}
-                    <p class="vads-u-font-weight--bold vads-u-font-family--serif">Available at</p>
+                    <h3>Available at these {{ regionNickname }} locations</h3>
```

Comment:
What was the reason for changing this to an h3?

Reply:
@accbjt i wish i knew :smile:  was part of the AC